pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2604,4 +2604,7 @@ in {
   purescript-ast = purescriptBrokenFlag (super.purescript-ast.overrideScope purescriptOverlay);
 
   purenix = super.purenix.overrideScope purescriptOverlay;
+
+  # tests use doctest-parallel which produces some errors during testing
```

Comment:
A couple things:

- Could you rebase this on the `haskell-updates` branch?  We generally ask that all haskell-related PRs go to that branch.
- Are you sure that these doctests are failing because of some Nix/sandbox related reason, and not because they are actually broken for some reason?  Do you know why they are broken?  Does it make sense to have an upstream issue tracking this?
- It looks like `clash-prelude` is in `pkgs/development/haskell-modules/configuration-hackage2nix/transitive-broken.yaml`.  Could you remove it from that list and confirm that it no longer depends on any packages that are marked broken?

Reply:
I did points 1 and 3. Regarding 2, I'm not really sure why the tests fail, it was not straightforward and I didn't have time to investigate.

Reply:
Thanks, this looks good!

Reply:
This should be available in `master` when https://github.com/NixOS/nixpkgs/pull/181822 is merged in.